### PR TITLE
Show message and redirect to Login after user creation

### PR DESCRIPTION
**Pull request by vatsa287**
Fixes: #30
Signed-off-by: Shree Vatsa N <vatsa@kadalu.tech>

### CODE REVIEW

File: templates/user_create.html.j2 [hunk 1]
```diff
@@ -9,6 +9,9 @@
                     <div class="box">
                         <article class="message is-danger" x-show="error != ''">
                             <div class="message-body" x-text="error"></div>
+                        </article>
+                         <article class="message is-success" x-show="showRedirectToPools">
```
Comment (aravindavk) [inline templates/user_create.html.j2:13]: Place this outside of above div. Show only this message.

File: templates/user_create.html.j2 [hunk 2]
```diff
@@ -9,6 +9,9 @@
                     <div class="box">
```
Comment (aravindavk) [inline templates/user_create.html.j2:9]: `x-show="message == ''"`

File: templates/user_create.html.j2 [hunk 3]
```diff
@@ -9,6 +9,9 @@
...
                         <article class="message is-danger" x-show="error != ''">
                             <div class="message-body" x-text="error"></div>
+                        </article>
+                         <article class="message is-success" x-show="showRedirectToPools">
+                            <div class="message-body" x-html="`<p>User Created Successfully! Login To Proceed.</p>`"></div>
```
[the remark this answers is not in the document]
Comment (vatsa287) [reply templates/user_create.html.j2]: Ack.

File: templates/user_create.html.j2 [hunk 4]
```diff
@@ -32,11 +35,17 @@
                                 </div>
                             </div>
 
-                            <div class="field has-text-centered">
+                            <div class="field has-text-centered" x-show="showCreateUserButton">
```
[the remark this answers is not in the document]
Comment (vatsa287) [reply templates/user_create.html.j2]: done

File: templates/user_create.html.j2 [hunk 5]
```diff
@@ -32,11 +35,17 @@
...
                                 <div class="control">
                                     <input type="submit" class="button is-primary" value="Create User"/>
                                 </div>
                             </div>
+
```
[the remark this answers is not in the document]
Comment (vatsa287) [reply templates/user_create.html.j2]: done

File: templates/user_create.html.j2 [hunk 6]
```diff
@@ -71,7 +82,14 @@
...
                      await mgr.createUser(this.username, this.password);
-                     location.href = `/pools?mgr=${this.mgrUrl}`;
+                     /*
+                        TODO: Previous error message still persists when page is not reloaded &
+                        user creation is successfull. Causing both error & success msg to
+                        be displayed. Set error to default.
+                     */
+                     this.error = "";
+                     this.showCreateUserButton = false;
+                     this.showRedirectToPools = true;
```
[the remark this answers is not in the document]
Comment (vatsa287) [reply templates/user_create.html.j2]: done

File: templates/user_create.html.j2 (hunk 1 above)
Comment (vatsa287) [reply templates/user_create.html.j2]: done

**Pull request merged**